data cleaning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Fill any missing state names in dataset
</pre><p>To be extra safe, we could consider running this on all datapoints</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>